rero_ils/modules/documents/dojson/contrib/jsontomarc21/model.py

```
@@ -683,6 +1226,85 @@ def reverse_physical_description(self, key, value):
     return result or None
 
 
+@to_marc21.over("500", "^general_notes")
+@utils.reverse_for_each_value
+@utils.ignore_value
+def reverse_general_notes(self, key, value):
```

Ruff (0.15.1)

[warning] Unused function argument: `self`

(ARG001)